.travis.yml

```diff
@@ -17,35 +17,35 @@ notifications:
     recipients:
 
 install:
-    # Pull in MoveIt's ROS Docker CI image (piping to `/dev/null` to hide output)
-    - DOCKER_IMAGE=moveit/moveit:$ROS_DISTRO-ci
-    - echo "Pulling in Docker Image \"$DOCKER_IMAGE\""
-    - docker pull $DOCKER_IMAGE > /dev/null
-    # Tests default to false if not set
-    - RUN_BUILD=${RUN_BUILD:-"false"}
-    - RUN_TESTS=${RUN_TESTS:-"false"}
-    - RUN_COVERAGE=${RUN_COVERAGE:-"false"}
-    - RUN_FORMATTING_CHECKS=${RUN_FORMATTING_CHECKS:-"false"}
-    # Run the CI script inside the docker container
-    # (set tests/checks to false by default)
-    - docker run 
-      --env RUN_BUILD
-      --env RUN_TESTS
-      --env RUN_COVERAGE
-      --env RUN_FORMATTING_CHECKS
-      --env ROS_DISTRO 
-      --env REPOSITORY_NAME
-      --env TRAVIS_PULL_REQUEST
-      --env TRAVIS_BRANCH
-      --volume $(pwd):/root/$REPOSITORY_NAME
-      $DOCKER_IMAGE
-      /bin/bash -c "/root/$REPOSITORY_NAME/travis_ci.sh"
+  # Pull in MoveIt's ROS Docker CI image (piping to `/dev/null` to hide output)
```

Comment:
This file should not be modified. You should watch out when committing to make sure you are committing only the files you want.